### ISSUE TITLE
CASES - Choose language of Case Template

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#472]
Hi, 

We communicate with clients from 4 countries, in each is different language. Will it be possible to choose the case language for case templates - like lang case create template and .... 

Or, if can be possible to set the contact default language and the case will know which lang template it can send to client/contact . This helps so much in everyday use of SuiteCRM 

Sorry for English 
Regars Mikolaj

**Comment by ghermans** [comment]
:+1:

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [CASES - Choose language of Case Template](https://trello.com/c/nX0fgfAh/148-cases-choose-language-of-case-template)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
